projects/melon-runtime/Commands/melon.js

```diff
@@ -28,11 +28,11 @@ if(!process.argv.includes("--ignore-update")) {
 }
 
 if(process.argv.includes("--ignore-update")) {
-    args = args.filter(x => !x.startsWith("--"));
+    args = args.filter(x => x != "--ignore-update");
 
     //Dotnet Melon implementation
     const outputDirectory = __dirname.replace('Commands', 'Output');
-    const melon = spawn('dotnet', ["exec", path.join(outputDirectory, "Melon.dll"), ...args], spawnOptions);
+    const melon = spawn('dotnet', ["exec", path.join(outputDirectory, "MelonRuntime.CLI.dll"), ...args], spawnOptions);
 
     melon.on('data', console.log);
 }
```

Comment:
Note: a micro rework is needed for "melon.js" npx command, probably a better, well-organised implementation to separate the **auto-update** feature and the common execution workflow. A note is needed in the internal project `MelonRuntime.CLI` about the `--ignore-update` instruction that is an external (npx only flag)